source/js/components/distance-filter.js

```diff
@@ -0,0 +1,42 @@
+import results from './results.js';
```

Comment:
[fix] It might be easiest to sit down and talk through this, but I think there's a better pattern you can use for organizing everything here to eliminate a lot of repeating code from your two `filter` files. Essentially, I would make your two filter files do nothing but pass values back to your `search-form.js` file, so all of the ajax submissions can be sent and received from one place.